src/applications/personalization/rated-disabilities/components/TotalRatingStates.jsx

```diff
@@ -53,34 +60,32 @@ export const missingTotalMessage = () => {
 };
 
 export const totalRatingMessage = totalDisabilityRating => (
-  <span>
-    <div className="total-rated-disabilities">
-      <div className="vads-l-row medium-screen:vads-u-padding-bottom--2p5">
-        <div className="vads-l-col--12 medium-screen:vads-l-col--6 small-desktop-screen:vads-l-col--9 vads-u-padding-right--1p5">
-          <p>
-            <strong>
-              Your combined disability rating is {totalDisabilityRating}%
-            </strong>
-            . This rating doesn't include disabilities for your claims that are
-            still pending. You can check the status of your disability claims or
-            appeals with the Claim Status tool.
-          </p>
-          <a
-            href="/claim-or-appeal-status/"
-            arial-label="check your claims or appeals status"
-            title="check your claims or appeals status"
-          >
-            Check your claims or appeals
-          </a>
-        </div>
-        <div className="vads-u-margin-top--2p5 medium-screen:vads-u-margin-top--0 vads-l-col--12 medium-screen:vads-l-col--6 small-desktop-screen:vads-l-col--3 medium-screen:vads-u-padding-top--2p5">
-          <PercetnageCalloutBox
-            value={totalDisabilityRating}
-            isPercentage
-            label="Your combined VA disability rating"
-          />
-        </div>
+  <div className="total-rated-disabilities">
+    <div className="vads-l-row medium-screen:vads-u-padding-bottom--2p5">
+      <div className="vads-l-col--12 medium-screen:vads-l-col--6 small-desktop-screen:vads-l-col--9 vads-u-padding-right--1p5">
+        <p>
+          <strong>
+            Your combined disability rating is {totalDisabilityRating}%
+          </strong>
+          . This rating doesn't include disabilities for your claims that are
```

Comment:
curly quote

Reply:
@erikphansen - is the curly quote actually something standardized across other areas of `vets-website`? That's a good bit of trivia for me to keep in mind going forward.